scripts/Muon/GUI/Common/home_instrument_widget/home_instrument_widget_view.py

```diff
@@ -0,0 +1,585 @@
+# -*- coding: utf8 -*-
+from __future__ import (absolute_import, division, print_function)
+
+from PyQt4 import QtGui, QtCore
+from PyQt4.QtCore import pyqtSignal as Signal
+
+from Muon.GUI.Common.muon_file_utils import allowed_instruments
+from Muon.GUI.Common.run_string_utils import valid_float_regex
+
+
+class InstrumentWidgetView(QtGui.QWidget):
+    dataChanged = Signal()
+
+    def __init__(self, parent=None):
+        super(InstrumentWidgetView, self).__init__(parent)
+
+        self.layout = QtGui.QGridLayout(self)
+
+        self._button_height = 40
+        self._cached_instrument = ["None", "None"]
+
+        self._button = QtGui.QMessageBox.Ok
+        self._message_box = QtGui.QMessageBox("Error in substitution", "ERROR !", QtGui.QMessageBox.Warning,
```

Comment:
is there a reason we are not using the muon message box?

Reply:
I don't think so no